src/parameterList.ts

```diff
@@ -18,7 +18,8 @@ export const parameterDescriptions = {
   keepEmptyParagraphs: 'Keep all paragraphs, even empty ones.',
   mwWikiPath: 'Mediawiki wiki base path (per default "/wiki/")',
   mwActionApiPath: 'Mediawiki API path (per default "/w/api.php")',
-  mwRestApiPath: 'Mediawiki Rest API path (per default "/api/rest_v1")',
+  mwRestApiPath: 'Wikimedia Rest API path (per default "/api/rest_v1")',
+  mwMediaWikiRestApiPath: 'Mediawiki Rest API path (per default "w/rest.php/v1/page/")',
```

Comment:
This is probably not the best naming, but I have to follow the convention that `mw` stands for Mediawiki singleton in mwoffliner, and `MediaWikiRestApiPath` should be named like this because this is how this API was named originally.